source/utils/Helper.h

```diff
@@ -30,6 +30,10 @@
 #include <vector>
 #include <cstdint>
 
+#ifdef _MSC_VER
+    #define snprintf _snprintf
```

Comment:
I've had a look and found how CEGUI and Ogre does. You can see it here:
https://github.com/OpenTechEngine/CEGUI/blob/master/cegui/src/DefaultLogger.cpp#L91
https://github.com/ehsan/ogre/blob/master/OgreMain/src/OgreLog.cpp#L91

I guess both are reliable and do more or less the same. IMHO, it would be wise to also do that. WDYT ?